src/pages/HomeNeedLeadView.tsx

```diff
@@ -58,20 +59,67 @@ const useStyles = makeStyles(() => ({
 }));
 
 const HomeNeedLeadView = () => {
-  // const { data: needsData } = useQuery(GET_NEEDS, {
-  //   variables: {
-  //     request: {
-  //       options: {
-  //         limit: 5
-  //       }
-  //     }
-  //   },
-  //   fetchPolicy: "network-only"
-  // });
-  // const needs = needsData?.getAllNeeds?.needs;
+  const { data: needsData } = useQuery(GET_NEEDS, {
+    variables: {
+      request: {
+        options: {
+          page: 0,
+          limit: 5
+        }
+      }
+    },
+    fetchPolicy: "network-only"
+  });
+
+  const { data: leadsData } = useQuery(GET_LEADS, {
+    variables: {
+      request: {
+        options: {
+          page: 0,
+          limit: 5
+        }
+      }
+    },
+    fetchPolicy: "network-only"
+  });
+
+  const needs = needsData?.getAllNeeds?.needs || needsDummy;
+  const leads = leadsData?.getAllLeads?.leads || leadsDummy;
 
```

Comment:
Mmmh whats the reason of doing that ? Is the query not working? If you are using the dummy for testing thats okey, otherwise we dont need it :)

Reply:
Just for my testing locally :D